security/security-design/oci-security-health-check-standard/README.md

```diff
@@ -0,0 +1,102 @@
+# OCI Security Health Check - Standard Edition
+
+Owner: Olaf Heimburger
+
+## When to use this asset?
```

Comment:
Added this per my template. I'm ok with it being empty for now, please at some point.